### PR TITLE
iOS SDK Documentation updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Fixed function name and combined the success tips to make them look less repetitive.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx [hunk 1]
```diff
@@ -14,12 +14,7 @@ Check out the [**GitHub codebase**](https://github.com/rudderlabs/rudder-sdk-ios
 
 <div class="successBlock">
 
-  Starting from version <strong>1.1.0</strong>, the iOS SDK supports the <b>tvOS</b> platform.  You can now integrate the SDK with your tvOS apps and seamlessly track the user events without the need for any additional configuration.
-</div>
-
-<div class="successBlock">
-
-  Starting from version <strong>1.3.0</strong>, the iOS SDK supports the <strong>watchOS</strong> platform.  You can now integrate the SDK with your watchOS apps and seamlessly track the user events without the need for any additional configuration.
+  Starting from version <strong>1.1.0</strong>, the iOS SDK supports the <strong>tvOS</strong> platform. RudderStack also supports the <strong>watchOS</strong> platform starting from version <strong>1.3.0</strong>. You can now integrate the iOS SDK with your tvOS and watchOS apps and seamlessly track user events without the need for any additional configuration.
```
Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx:17]: Version should be 1.3.1

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx [hunk 2]
```diff
@@ -1007,7 +1002,7 @@ Update the usage of the following classes as per the table below:
 
 ### How do I ensure the events tracked just before closing/backgrounding the app are sent immediately and not on the next app launch?
 
-To ensure that the events tracked just before closing/backgrounding the app are sent to RudderStack immediately, you can set `withBackgroundMode` to `YES` while creating the `RSConfig` object as shown below:
+To ensure that the events tracked just before closing/backgrounding the app are sent to RudderStack immediately, you can set `withEnableBackgroundMode` to `YES` while creating the `RSConfig` object as shown below:
```
Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx:1005]: Can you change to `RSConfigBuilder` from `RSConfig`?

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx:17]: got it - will update!

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk/index.mdx:1005]: Sure

**Review by pallabmaiti — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
All the suggestions are incorporated.

**Pull request merged**